zebra/connected.c

```diff
@@ -251,11 +252,16 @@ void connected_up(struct interface *ifp, struct connected *ifc)
 
 	metric = (ifc->metric < (uint32_t)METRIC_MAX) ?
 				ifc->metric : ifp->metric;
-	rib_add(afi, SAFI_UNICAST, zvrf->vrf->vrf_id, ZEBRA_ROUTE_CONNECT,
-		0, 0, &p, NULL, &nh, 0, zvrf->table_id, metric, 0, 0, 0);
 
-	rib_add(afi, SAFI_MULTICAST, zvrf->vrf->vrf_id, ZEBRA_ROUTE_CONNECT,
-		0, 0, &p, NULL, &nh, 0, zvrf->table_id, metric, 0, 0, 0);
+	if (CHECK_FLAG(ifc->conf, ZEBRA_IFC_QUEUED)
+	    && !CHECK_FLAG(ifc->conf, ZEBRA_IFC_REAL))
+		flags |= ZEBRA_FLAG_NEVER_SELECT;
```

Comment:
The route is independent from the address;  DAD failure doesn't make the route unusable.  If we do weird things with the route, we can get desynced between what FRR expects to happen & what actually happens in forwarding...